tasks.py

```diff
@@ -0,0 +1,83 @@
+from robocorp.tasks import task
+from robocorp import browser, http, excel
+from RPA.PDF import PDF
+
+@task
+def robot_spare_bin_python():
+    """Robot to enter weekly sales data into the RobotSpareBin Industries Intranet."""
+
+    # Without this, nothing is visible as it is running headless
+    browser.configure(
+        slowmo=100,
```

Comment:
This makes it slow for demo purposes, but not sure if we want it here to complicate things?

Reply:
for beginners it might be helpful to see slowly, step by step what is happening.
It's also to raise awareness that this option exists.